### DISCUSSION TITLE
why Safari can't open Metamask，It's always going to the appstore

### DISCUSSION THREAD

**Discussion by 0x-sen** [WalletConnect/walletconnect-monorepo#1140] [Developer Support]
https://example.walletconnect.org/
<img width="470" alt="image" src="https://user-images.githubusercontent.com/30768615/171867092-165fb169-1a51-4609-bc58-c313068f2f14.png">
<img width="329" alt="image" src="https://user-images.githubusercontent.com/30768615/171867295-fa2daa0f-d548-4b36-a025-b762642a26cf.png">

**Comment by vphilot** [comment]
same issue here, not sure if related to WC or the MM app?

**Comment by chadyj** [comment]
I couldn't reproduce this.
Could you try deleting Metamask, and re-install v5.2.0?

**Comment by SuperCipher** [comment]
For me, this seems to happen when I try to do personal signs using a metamask.
